### DISCUSSION TITLE
New Runelite won't show you your downloaded plugins

### DISCUSSION THREAD

**Discussion by asad-bashi** [runelite/runelite#17265] [Q&A] [answered]
Hello 

I'm using runelite for the first time since last year and I think the newest update messed something up. Going into the plugin hub and downloaded it doesn't show up in the configuration list. 

Here is a example of a plugin i want to use.

![plugin1](https://github.com/runelite/runelite/assets/93698197/bac22a00-5419-4d79-ad87-c76681221813)
![plugin2](https://github.com/runelite/runelite/assets/93698197/46f6cc3a-9bea-45d6-b0b5-40fa3589430f)

Not just with this plugin but any new ones I want to download don't show up. 

I tried re installing the client but it's still not working

**Comment by Felanbird** [comment, marked as answer]
Github is not a support forum, join the discord for support to have this issue resolved
![221373292-659c7b87-eef2-400b-95e3-e1974c6535a9](https://github.com/runelite/runelite/assets/41973452/ad9ff9f5-6d9b-4490-8276-cfe3f8bca29e)
